lib/modules/manager/custom/regex/readme.md

```diff
@@ -88,15 +87,13 @@ But you don't want to write a regex custom manager rule for _each_ variable.
 Instead you enhance your `Dockerfile` like this:
 
 ```Dockerfile
-ARG IMAGE=node:12@sha256:6e5264cd4cfaefd7174b2bc10c7f9a1c2b99d98d127fc57a802d264da9fb43bd
-FROM ${IMAGE}
-# renovate: datasource=github-tags depName=nodejs/node versioning=node
-ENV NODE_VERSION=10.19.0
-# renovate: datasource=github-releases depName=composer/composer
+# renovate: datasource=github-tags depName=node packageName=nodejs/node versioning=node
+ENV NODE_VERSION=20.10.0
+# renovate: datasource=github-releases depName=composer packageName=composer/composer
 ENV COMPOSER_VERSION=1.9.3
-# renovate: datasource=docker depName=docker versioning=docker
+# renovate: datasource=docker packageName=docker versioning=docker
 ENV DOCKER_VERSION=19.03.1
-# renovate: datasource=npm depName=yarn
+# renovate: datasource=npm packageName=yarn
```

Comment:
```suggestion
# renovate: datasource=npm depName=YARN packageName=yarn
```
Example of using both.

Reply:
There's already two examples earlier so I preferred to have this one without